libraries/lib-strings/UnusedStrings.h

```diff
@@ -96,25 +96,6 @@ Some example strings are also given first, to document the syntax.
 // //////////////////////////////////////////////// End examples
 #endif
 
-
-// Crash & error report window
-
-XO("Problem Report for Audacity"),
-XO("Click \"Send\" to submit the report to Audacity. This information is collected anonymously."),
-XO("Problem details"),
-XO("Comments"),
-XC("&Send",       "crash reporter button"),
-XC("&Don't send", "crash reporter button"),
-// i18n-hint C++ programming exception
-XO("Exception code 0x%x"),
```

Comment:
Preservation of the i18n-hint comments is still not done after the recent force push.